tests/unauthorized/appsTest.js

```diff
@@ -24,7 +24,7 @@ describe('Unauthorized: Currently testing apps management,', function () {
     oc.login()
   })
 
-  it('checking method : getApps', function (done) {
+  fit('checking method : getApps', function (done) {
```

Comment:
I believe its enough to have `fdescribe` on the top if you want to enable all the tests

Reply:
fixed